paydb_endpoint.py

```diff
@@ -288,6 +311,30 @@ def user_update_email_confirm(token=None):
     flash('User email updated.', 'success')
     return redirect('/')
 
+@paydb.route('/user_update_password', methods=['GET', 'POST'])
+def user_update_password():
+    sig = request_get_signature()
+    content = request.get_json(force=True)
+    if content is None:
+        return bad_request(web_utils.INVALID_JSON)
+    params, err_response = get_json_params(content, ["api_key", "nonce", "current_password", "new_password"])
+    if err_response:
+        return err_response
+    api_key, nonce, current_password, new_password = params
+    res, reason, api_key = check_auth(db.session, api_key, nonce, sig, request.data)
+    if not res:
+        return bad_request(reason)
+    user = api_key.user
+    verifying_password = verify_password(current_password, user.password)
```

Comment:
rename `verifying_password` to `verified_password` or `password_ok`

Reply:
done